src/fe/inf_fe_map.C

```diff
@@ -203,11 +203,20 @@ Point InfFEMap::inverse_map (const unsigned int dim,
       libmesh_error_msg("Invalid dim = " << dim);
     }
 
+#ifdef DEBUG
+  // In debug mode, the validity of base_elems id is checked.
+  // Thus, we should set it to a valid  one:
+  base_elem->set_id(inf_elem->id());
+#endif
+
   // 3.)
   // Now we have the intersection-point (projection of physical point onto base-element).
   // Lets compute its internal coordinates (being p(0) and p(1)):
   p= FEMap::inverse_map(dim-1, base_elem.get(), intersection,
                         tolerance, secure);
+#ifdef DEBUG
+  base_elem->invalidate_id();
+#endif
```

Comment:
This is a temporary proxy Elem that's getting deleted, isn't it?  So do we need to bother invalidating its id?

Reply:
... I wanted to be accurate.
But I agree, we can skip it.